roles/openshift_logging/tasks/scale.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- shell: >
+    {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig get {{object}}
+    --template='{{ '{{.spec.replicas}}' }}' -n {{openshift_logging_namespace}}
```

Comment:
s/shell/command/

consider using `-o jsonpath` instead of `--template` to avoid having to quote braces.